collector/generatorreceiver/topos/hipster_shop.yaml

```diff
@@ -113,9 +113,9 @@ topology:
           tags:
             host.name: frontend.internal
             cpu: "1"
-            namespace: hipster-shop  
+            namespace: hipster-shop
       routes:
-        - route: /product
+        /product:
```

Comment:
I would not have thought this would work! The slash in the key, I mean. It looks super weird! But I guess basically all printable characters are allowed. Neat!

Reply:
I was also surprised that it worked!